react_app/src/components/HomePage/AddCourseFormDialog.js

```diff
@@ -38,7 +44,48 @@ function AddCourseFormDialog(props) {
         <DialogTitle id="form-dialog-title">Add Course</DialogTitle>
         <DialogContent>
           <DialogContentText>
-            To add course, please enter details below:
+            For Admins: To add course, please enter details below:
```

Comment:
`{userRole === 'Admin' ? For Admins: To add course, please enter details below :  Don't see the course you're looking for? Send us the course details and we'll add it for you!}` ?

Reply:
good point! I've refactored it with these conditional ternary operators 👍